arches_lingo/src/arches_lingo/components/concept/ConceptImages/components/ConceptImagesViewer.vue

```diff
@@ -0,0 +1,195 @@
+<script setup lang="ts">
+import { inject, ref, onMounted } from "vue";
+import { useGettext } from "vue3-gettext";
+
+import Button from "primevue/button";
+
+import FileListWidget from "@/arches_component_lab/widgets/FileListWidget/FileListWidget.vue";
+
+import { DANGER, SECONDARY, VIEW } from "@/arches_lingo/constants.ts";
+import { useConfirm } from "primevue/useconfirm";
+
+import type {
+    ConceptImages,
+    DigitalObjectInstance,
+} from "@/arches_lingo/types.ts";
+import { fetchLingoResource } from "@/arches_lingo/api.ts";
+import NonLocalizedStringWidget from "@/arches_component_lab/widgets/NonLocalizedStringWidget/NonLocalizedStringWidget.vue";
+
+const props = defineProps<{
+    tileData: ConceptImages | undefined;
+    componentName: string;
+    sectionTitle: string;
+    graphSlug: string;
+    nodegroupAlias: string;
+}>();
+
+const resources = ref<DigitalObjectInstance[]>();
+const { $gettext } = useGettext();
+const confirm = useConfirm();
+
+onMounted(async () => {
+    if (props.tileData) {
+        resources.value = await Promise.all(
+            props.tileData.aliased_data.depicting_digital_asset_internal.map(
```

Comment:
IMO `props.tileData` should be read-only for clarity. mind looping this into a separate ref?

Reply:
I don't follow here - are you seeing me mutating props.tileData somewhere?